modules/ROOT/pages/metadata-keys.adoc

```diff
@@ -86,6 +87,49 @@ public class DocsMetadataSource extends Source<Map<String, Object>, Void> {
   }
 ----
 
+[NOTE]
+The example described above is the only way to support metadata keys for sources. Do not set the resolver at the `@MetadataKeyId` annotation when defining sources.
+
+Setting the `@MetadataScope` annotation at the operation class definition level indicates that all of its operations provide metadata and that they must all have a parameter annotated with `@MetadataKeyId` while they inherit the resolver from the scope.
```

Comment:
```suggestion
Setting the `@MetadataScope` annotation at the operation class definition level indicates that all of its operations provide metadata and that they must all have a parameter annotated with `@MetadataKeyId` because they inherit the resolver from the scope.
```

Reply:
?

Reply:
Dev confirmed this makes sense and does not change the meaning.